tests/Feature/app/Models/UserCompanyTest.php

```
@@ -227,8 +227,8 @@
 
         $this->assertCount(2, $usersPerCompany);
 
-        $activeCompanyUserCount = $usersPerCompany->where('company_id', $activeCompany->uuid)->first();
-        $inactiveCompanyUserCount = $usersPerCompany->where('company_id', $inactiveCompany->uuid)->first();
+        $activeCompanyUserCount = $usersPerCompany->where('company_id', $activeCompany->id)->first();
```

PHPMD (2.15.0)

Avoid excessively long variable names like $activeCompanyUserCount. Keep variable name length under 20. (Naming Rules)

(LongVariable)